apps/dg/libraries/jquery/jquery.event.drag-2.2.js

```diff
@@ -37,6 +37,7 @@ drag = $special.drag = {
 		not: ':input', // selector to suppress dragging on target elements
 		handle: null, // selector to match handle target elements
 		relative: false, // true to use "position", false to use "offset"
+		touch: true, // [CC] false to suppress touch dragging, true to allow
```

Comment:
Can you elaborate on this change? It looks like you undo the effect of it in the Slick Grid code.

Reply:
Never mind. On reflection, I understand. This is a _new_ default. Makes sense, now.

Reply:
It seemed like the right approach from the `jquery.event.drag` perspective would be to support touch by default but then to allow clients to disable it if desired.

Reply:
I agree. I had just misunderstood it, initially.